File: exercises/gh-copilot-refactor/gh-copilot-refactor-js/refactor/builder/builder.js
class House {
  constructor(windows, doors, floors) {
    this.windows = windows;
    this.doors = doors;
    this.floors = floors;
  }
}

function buildHouse(windows, doors, floors) {
  return new House(windows, doors, floors);
}

const house = buildHouse(10, 2, 3);
console.log(`House: ${JSON.stringify(house)}`);

File: exercises/gh-copilot-refactor/gh-copilot-refactor-java/copilot-refactor/src/main/java/com/ttpsc/copilot_training/builder/Builder.java
package com.ttpsc.copilot_training.builder;

public class Builder {

    static class House {
        int windows;
        int doors;
        int floors;

        House(int windows, int doors, int floors) {
            this.windows = windows;
            this.doors = doors;
            this.floors = floors;
        }

        @Override
        public String toString() {
            return "House{" +
                    "windows=" + windows +
                    ", doors=" + doors +
                    ", floors=" + floors +
                    '}';
        }
    }

    public static House buildHouse(int windows, int doors, int floors) {
        return new House(windows, doors, floors);
    }

    public static void main(String[] args) {
        House house = buildHouse(10, 2, 3);
        System.out.println("House: " + house);
    }
}